analyzers/tests/SonarAnalyzer.UnitTest/Rules/MemberShouldNotHaveConflictingTransparencyAttributesTest.cs

```diff
@@ -25,18 +25,23 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class MemberShouldNotHaveConflictingTransparencyAttributesTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<MemberShouldNotHaveConflictingTransparencyAttributes>();
+
         [TestMethod]
         public void MemberShouldNotHaveConflictingTransparencyAttributes() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\MemberShouldNotHaveConflictingTransparencyAttributes.cs", new MemberShouldNotHaveConflictingTransparencyAttributes());
+            builder.AddPaths("MemberShouldNotHaveConflictingTransparencyAttributes.cs").Verify();
 
         [TestMethod]
         public void MemberShouldNotHaveConflictingTransparencyAttributes_AssemblyLevel() =>
-            OldVerifier.VerifyNonConcurrentAnalyzer(@"TestCases\MemberShouldNotHaveConflictingTransparencyAttributes_AssemblyLevel.cs", new MemberShouldNotHaveConflictingTransparencyAttributes());
+            builder.AddPaths("MemberShouldNotHaveConflictingTransparencyAttributes_AssemblyLevel.cs").WithConcurrentAnalysis(false).Verify();
 
 #if NET
+
         [TestMethod]
         public void MemberShouldNotHaveConflictingTransparencyAttributes_CSharp10() =>
-            OldVerifier.VerifyAnalyzerFromCSharp10Library(@"TestCases\MemberShouldNotHaveConflictingTransparencyAttributes.CSharp10.cs", new MemberShouldNotHaveConflictingTransparencyAttributes());
+            builder.AddPaths("MemberShouldNotHaveConflictingTransparencyAttributes.CSharp10.cs").WithConcurrentAnalysis(false).WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
```

Comment:
Do we only put invocations in separate lines if S103 is violated (that is what the coding style doc says)? My understanding as of now was that it was only a single line chain if there was nothing done except add path and verify.

Reply:
I think we easily get used to reading this kind of test and having them on the same line should be acceptable. I'm also perfectly fine with a single invocation per line but sometimes I feel it doesn't help much.